### DISCUSSION TITLE
Support within the "link" field and entry linking within navs

### DISCUSSION THREAD

**Discussion by royduin** [statamic-rad-pack/runway#271] [Feature Requests]
It would be nice if we could select Runway resources from the "link" field and when linking entries in a navigation.

For the time being I'm using the belong to relation next to the link field and within the navigation blueprint. But this results in this kind of things in templates:
```
<a href="{{ isset($item['product']->value()['url']) ? '/'.$item['product']->value()['url'] : $item['url'] }}">
```

**Comment by duncanmcclean** [comment]
Unfortunately, this isn't possible to build right now as there's no way to extend upon the options in the Link Fieldtype (URL / entries / assets).

See code: https://github.com/statamic/cms/blob/4.x/resources/js/components/fieldtypes/LinkFieldtype.vue#L21

Although, if Statamic adds supports for addons to extend the options, I'd consider adding it. For now though, I'm closing this feature request.

**Comment by duncanmcclean** [comment]
This will be possible soon. "Subscribe" to this PR for updates when its merged & tagged: https://github.com/statamic-rad-pack/runway/pull/838
